src/styles/index.scss

```diff
@@ -2,2 +1,231 @@
+  &::after {
+    content: '';
+    background-color: var(--main-text-color);
+    border-radius: 8px;
+    width: 100%;
+    height: 4px;
+    display: block;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.card {
+  width: 200px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition-duration: 300ms;
+
+  &__img {
+    background-position: center;
+    background-size: cover;
+    margin: 33px 20px 0;
+  }
+
+  &__title {
+    display: block;
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 17px 0;
+    color: rgba(6, 11, 53, 1);
+    height: 36px;
+  }
+
+  &__code {
+    display: block;
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: rgba(97, 96, 112, 1);
+    margin: 4px 0 0 17px;
+    height: 14px;
+  }
+
+  &__review {
+    display: flex;
+    width: 166px;
+    justify-content: space-between;
+    margin: 17px 17px 0;
+    align-items: center;
+    height: 16px;
+  }
+
+  &__review-amount {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    color: rgba(6, 11, 53, 1);
+    margin-left: 16px;
+  }
+
+  &__price {
+    display: flex;
+    margin: 24px 17px 0;
+    justify-content: space-between;
+    align-items: center;
+    width: 166px;
+    height: 18px;
+  }
+
+  &-title {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &-amount {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__buy {
+    display: flex;
+    width: 166px;
+    height: 40px;
+    margin: 16px;
+    background-color: rgba(0, 172, 220, 1);
+    border-radius: 5px;
+    border: none;
+    align-items: center;
+    justify-content: center;
+    text-decoration: none;
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    color: rgba(255, 255, 255, 1);
+    text-transform: uppercase;
+  }
+
+  &__buy:hover {
+    background-color: rgba(255, 255, 255, 1);
+    border: 1px solid rgba(0, 172, 220, 1);
+    color: rgba(0, 172, 220, 1);
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
+  .card__title {
+    color: #34568b;
+  }
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star:not(:first-child) {
+    margin-left: 4px;
+  }
+
+  &__star:nth-child(5) {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
This selector could potentially lead to an issue. If the number of active stars changes, this style will only be applied to the 5th star. It would be better to create a separate class for inactive stars and apply the different background image to that class.